meta-bases/azure-base/autobuild/build

```diff
@@ -0,0 +1,15 @@
+mkdir "$PKGDIR"
```

Comment:
Verbosity here.

Reply:
There is also no way that the `catdoc` below would work, since you never created the subdirectories.